content/en/serverless/configuration/_index.md

```diff
@@ -606,6 +618,10 @@ To migrate, compare the [installation instructions using the Datadog Lambda Exte
 {{% /tab %}}
 {{< /tabs >}}
 
+## Configure the Datadog Lambda extension for local testing
+
+If you test your Lambda function's container image locally with the Datadog Lambda extension installed, you need to set `DD_LOCAL_TEST` to `true` in your local testing environment, otherwise the extension waits for responses from the AWS Extensions API and blocks the invocation.
```

Comment:
```suggestion
To test your Lambda function's container image locally with the Datadog Lambda extension installed, you need to set `DD_LOCAL_TEST` to `true` in your local testing environment. Otherwise, the extension waits for responses from the AWS Extensions API and blocks the invocation.
```